pandas/tests/reductions/test_reductions.py

```diff
@@ -964,25 +964,27 @@ def test_any_all_object_dtype_missing(self, data, bool_agg_func):
         expected = bool_agg_func == "any" and None not in data
         assert result == expected
 
+    @pytest.mark.parametrize("dtype", ["boolean", "Int64", "UInt64", "Float64"])
     @pytest.mark.parametrize("bool_agg_func", ["any", "all"])
     @pytest.mark.parametrize("skipna", [True, False])
     @pytest.mark.parametrize(
         # expected_data indexed as [[skipna=False/any, skipna=False/all],
         #                           [skipna=True/any, skipna=True/all]]
         "data,expected_data",
         [
-            ([False, False, False], [[False, False], [False, False]]),
-            ([True, True, True], [[True, True], [True, True]]),
-            ([pd.NA, pd.NA, pd.NA], [[pd.NA, pd.NA], [False, True]]),
-            ([False, pd.NA, False], [[pd.NA, False], [False, False]]),
-            ([True, pd.NA, True], [[True, pd.NA], [True, True]]),
-            ([True, pd.NA, False], [[True, False], [True, False]]),
+            ([0, 0, 0], [[0, 0], [0, 0]]),
```

Comment:
why do these return ints rather than True/False?

Reply:
Thanks, that was a mistake - it actually returns bools, test just happened to work because the reductions return a scalar and 1 == True, etc